ui/packages/catalog-ui-search/src/main/webapp/component/list-create/list-create.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Connexta, LLC
+ *
+ * <p>Unlimited Government Rights (FAR Subpart 27.4) Government right to use, disclose, reproduce,
+ * prepare derivative works, distribute copies to the public, and perform and display publicly, in
+ * any manner and for any purpose, and to have or permit others to do so.
+ */
+import React from 'react'
+import ListCreate from '../../react-component/presentation/list-create/list-create'
+import withListenTo from 'catalog-ui-search/src/main/webapp/react-component/container/backbone-container'
+
+var Marionette = require('marionette')
+
+const ListCreateWithListenTo = withListenTo(ListCreate)
+
+const ListCreateView = Marionette.LayoutView.extend({
+  template() {
+    return (
+      <ListCreateWithListenTo
+        model={this.model}
```

Comment:
It doesn't look like you're using this prop in your component. Can it be removed?

Reply:
It gets used here: https://github.com/codice/ddf/pull/4536/files/a951f1e141f4c8edace811e309c7e505a1290c29#diff-a85b0f2201e11c353e46dd9aeeb5bfdcR50